conans/test/utils/tools.py

```diff
@@ -306,6 +314,60 @@ def create_local_git_repo(files=None, branch=None, submodules=None, folder=None)
     return tmp.replace("\\", "/"), git.get_revision()
 
 
+def handleRemoveReadonly(func, path, exc):  # TODO: May promote to conan tools?
```

Comment:
yes please, this often fails for me in various situations using conan commands with git repositories, so had to do removes manually

Reply:
Could we issue a ticket for it? It would be nice to get this code review into the develop branch...

Reply:
Yes, please, maybe open an issue or directly a PR on top on this.